app/Composers/StatusPageComposer.php

```diff
@@ -62,9 +64,100 @@ public function compose(View $view)
         $componentGroups = ComponentGroup::whereIn('id', $usedComponentGroups)->orderBy('order')->get();
         $ungroupedComponents = Component::enabled()->where('group_id', 0)->orderBy('order')->orderBy('created_at')->get();
 
+        // Downtime stats for uptime table
+        $availability = array();
+
+
+        $now = Carbon::now();
+        $startOfDay=clone $now;
+        $startOfDay->setTime(0,0,0);
+        $endOfDay=clone $now;
+        $endOfDay->setTime(23,59,59);
+        $rangeToday = array();
+        array_push($rangeToday, $startOfDay);
+        array_push($rangeToday, $endOfDay);
+
+        //Today
+        $downtimeCollectionToday = Downtime::whereBetween('created_at', [$startOfDay, $endOfDay])
+                                            ->orWhereBetween('resolved_at', [$startOfDay, $endOfDay])
+                                            ->get();
+        array_push($availability, $this->calculateAvailability($downtimeCollectionToday, $rangeToday));
+
+
+        // Last week
+        $last7Days = array();
```

Comment:
You use this array just to pass two arguments to function calculateAvailability. Just use two variables $sevenDaysAgo and $endOfDay (maybe rename this variable to $endOfToday). An array $last7Days means to me that we have seven days from index 0 to 6 in it.